test/includes/microcloud.sh

```diff
@@ -634,17 +634,17 @@ validate_system_lxd() {
     if [ -n "${profile_pool}" ]; then
       lxc profile device get default root pool | grep -q "${profile_pool}"
     elif [ "${has_microceph}" = 1 ] && [ "${remote_disks}" -gt 0 ] ; then
-       lxc profile device get default root pool | grep -q "remote"
+      lxc profile device get default root pool | grep -q "remote"
     elif [ -n "${local_disk}" ] ; then
-       lxc profile device get default root pool | grep -q "local"
+      lxc profile device get default root pool | grep -q "local"
     else
-       ! lxc profile device list default | grep -q "root" || false
+      ! lxc profile device list default | grep -q "root" || false
     fi
 
     if [ "${has_microovn}" = 1 ] && [ -n "${ovn_interface}" ] ; then
-       lxc profile device get default eth0 network | grep -q "default"
+      lxc profile device get default eth0 network | grep -q "default"
     else
-       lxc profile device get default eth0 network | grep -q "lxdfan0"
+      lxc profile device get default eth0 network | grep -q "lxdfan0"
```

Comment:
Similar to previous reviews, it would be good to try and update these so that we are testing for an exact match. Not urgent though.